workers/semantics/multiple-workers/006.html

```diff
@@ -2,6 +2,9 @@
 if (location.hash == '#1') {
   onconnect = function(e) {
     var port = e.ports[0];
+    onerror = e => {
+      port.postMessage(String(e))
+    }
```

Comment:
If `#1` has an error, won't that trigger `w1.onerror`?

Reply:
Not when it's a SharedWorker, in chromium, I found out. Also the spec has a difference between dedicated and shared workers: https://html.spec.whatwg.org/multipage/workers.html#runtime-script-errors-2

Reply:
Got it